src/sentry/static/sentry/app/components/userMisery.tsx

```diff
@@ -16,7 +16,7 @@ type Props = {
 function UserMisery(props: Props) {
   const {bars, barHeight, miserableUsers, miseryLimit, totalUsers} = props;
 
-  const palette = new Array(bars).fill(theme.purple300);
+  const palette = new Array(bars).fill([CHART_PALETTE[0][0]]);
```

Comment:
I understand why we're doing this but I really don't like how the colour becomes unknown :(

Reply:
Noted - not sure how to fix that